app/controllers/nodes_controller.rb

```diff
@@ -1,12 +1,22 @@
 class NodesController < ApplicationController
   def get_all_nodes
+
+    context_id = params[:context_id]
+
+    if (not context_id.nil?)
```

Comment:
This can be said without double negation as `if context_id.present?`. Typically parentheses are only used in conditions when there are multiple logical operators and you need to control precedence.